### PR TITLE
:running: Update Go 1.13

### PR DESCRIPTION
**Pull request by vincepri**
Signed-off-by: Vince Prignano <vincepri@vmware.com>

### CODE REVIEW

**Comment by vincepri** [comment]
Waiting on https://github.com/kubernetes/test-infra/pull/15620

**Comment by vincepri** [comment]
/assign @detiber 

I'll follow-up with another PR for pprof

File: Dockerfile [hunk 1]
```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 # Build the manager binary
-FROM golang:1.12.10 as builder
+FROM golang:1.13.4 as builder
```
Comment (detiber) [inline Dockerfile]: Should this be 1.13.5 instead?

File: config/crd/bases/infrastructure.cluster.x-k8s.io_awsclusters.yaml [hunk 2]
```diff
@@ -16,7 +16,6 @@ spec:
     listKind: AWSClusterList
     plural: awsclusters
     singular: awscluster
-  preserveUnknownFields: false
```
Comment (detiber) [inline config/crd/bases/infrastructure.cluster.x-k8s.io_awsclusters.yaml]: I think this needs an update to the Makefile to ensure we don't clobber this: https://github.com/kubernetes-sigs/cluster-api/blob/546d56bea2b8cd9ac24d90ed7599d209dfe2ccc5/Makefile#L187

**Comment by vincepri** [comment]
Should be good now!

**Comment by detiber** [comment]
@vincepri this still appears to be missing the Makefile update to avoid removing `preserveUnknownFields` from the generated CRDs.

**Comment by vincepri** [comment]
Whoops, forgot to push, done now!

File: config/infrastructure.cluster.x-k8s.io_awsclusters.yaml [hunk 3]
```diff
@@ -0,0 +1,880 @@
+
```
Comment (detiber) [inline config/infrastructure.cluster.x-k8s.io_awsclusters.yaml]: Accidental add?

File: config/infrastructure.cluster.x-k8s.io_awsmachines.yaml [hunk 4]
```diff
@@ -0,0 +1,532 @@
+
```
Comment (detiber) [inline config/infrastructure.cluster.x-k8s.io_awsmachines.yaml]: Accidental add?

File: config/infrastructure.cluster.x-k8s.io_awsmachinetemplates.yaml [hunk 5]
```diff
@@ -0,0 +1,443 @@
+
```
Comment (detiber) [inline config/infrastructure.cluster.x-k8s.io_awsmachinetemplates.yaml]: Accidental add?

File: config/infrastructure.cluster.x-k8s.io_awsclusters.yaml (hunk 3 above)
Comment (vincepri) [reply config/infrastructure.cluster.x-k8s.io_awsclusters.yaml]: Yeah

**Pull request merged**